### PR TITLE
Add typesafe swagger description module

### PR DESCRIPTION
**Pull request by little-inferno**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -312,6 +314,17 @@ lazy val swaggerUI =
       buildInfoPackage := "ru.tinkoff.tschema.swagger"
     )
 
+lazy val swaggerTypesafe = project
+  .in(file("modules/swaggerTypesafe"))
+  .dependsOn(kernel, swagger)
+  .settings(
+    commonSettings,
+    simulacrumSettings,
```
Comment (REDNBLACK) [inline build.sbt:322]: Why would we need simulacrum in there?

Comment (REDNBLACK) [reply build.sbt:322]: Same goes for macro-paradise

File: modules/swaggerTypesafe/src/test/scala/ru/tinkoff/tschema/swagger/typesafe/TypesafeDescriptionSpec.scala [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import TypesafeDescriptionSpec.api
+import com.typesafe.config.{ConfigFactory => TypesafeConfigFactory}
+import derevo.derive
+import io.circe.Printer
+import io.circe.syntax._
+import ru.tinkoff.tschema.swagger.{MkSwagger, Swagger, SwaggerObject}
+import ru.tinkoff.tschema.swagger.OpenApi.Method
+
+class TypesafeDescriptionSpec extends AnyFlatSpec {
+  val config  = TypesafeConfigFactory.load("swagger.conf")
```
Comment (REDNBLACK) [inline modules/swaggerTypesafe/src/test/scala/ru/tinkoff/tschema/swagger/typesafe/TypesafeDescriptionSpec.scala:15]: Is it config though?

**Review by REDNBLACK — CHANGES_REQUESTED** [review]

**Review by REDNBLACK — APPROVED** [review]

File: modules/swaggerTypesafe/src/test/scala/ru/tinkoff/tschema/swagger/typesafe/TypesafeDescriptionSpec.scala (hunk 2 above)
Comment (little-inferno) [reply modules/swaggerTypesafe/src/test/scala/ru/tinkoff/tschema/swagger/typesafe/TypesafeDescriptionSpec.scala]: removed

File: build.sbt (hunk 1 above)
Comment (little-inferno) [reply build.sbt:322]: To run tests on Scala 2.12

**Pull request merged**